indexer/main.go

```diff
@@ -155,9 +163,10 @@ func getLastProcessedHeight() (uint64, error) {
 	return height, err
 }
 
-// getCelestiaHeight retrieves the Celestia height for a given Ethereum block number
-func getCelestiaHeight(ethBlockNum uint64) (int64, bool, error) {
-	var celestiaHeight int64
+// getMapping retrieves the Celestia height, blob index for a given Ethereum block number
```

Comment:
where is the blob index returned? Looks like this function returns:
1. celestia height
2. blob commitment
3. bool representing if found

[uber nit] instead of doing 

```go
func getMapping(ethBlockNum uint64) (uint64, []byte, bool, error) {
	var height uint64
	var commitment []byte
	var found bool
```

we could do

```go
func getMapping(ethBlockNum uint64) (celestiaHeight uint64, blobCommitment []byte, isFound bool, err error) {
```

which converts a lot of the comment into code

Reply:
Fixed.